t/t2240-queue-cmd.t

```diff
@@ -11,37 +11,37 @@ LIST_JOBS=${FLUX_BUILD_DIR}/t/job-manager/list-jobs
 
 test_expect_success 'flux-queue: unknown sub-command fails with usage message' '
 	test_must_fail flux queue wrongsubcmd 2>usage.out &&
-	grep Usage: usage.out
+	grep usage: usage.out
```

Comment:
Could `grep -i` be used here and this test moved previous to the change in output to avoid a commit that breaks tests?

Reply:
oh good idea!